### PR TITLE
add alphashape

### PR DESCRIPTION
**Pull request by ngodber**
@conda-forge/help-python

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ngodber** [comment]
I'm happy to maintain this.

**Comment by ngodber** [comment]
@conda-forge/help-python the CI checks failing are due to:
conda.exceptions.ResolvePackageNotFound: 
  - click_log[version='>=0.3.2']

which I'm a bit confused about because 

https://anaconda.org/conda-forge/click-log

Anyway, my understanding is this is not a blocking issue?

File: recipes/alphashape/LICENSE [hunk 1]
```diff
@@ -0,0 +1,14 @@
+The MIT License (MIT)
+Copyright � 2019 Lilian Besson (Naereen), https://bitbucket.org/lbesson/ <naereen at crans dot org>
```
Comment (npavlovikj) [inline recipes/alphashape/LICENSE]: Looks like the license file is part of the pypi tarball, so please remove it from here.

File: recipes/alphashape/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  host:
+    - click >=6.0
+    - click_log >=0.3.2
+    - pip
+    - python
+    - scipy >=1.0.0
+    - shapely >=1.4.0
+  run:
+    - click >=6.0
+    - click_log >=0.3.2
```
Comment (npavlovikj) [inline recipes/alphashape/meta.yaml]: The name of the conda package is `click-log`, and not `click_log`, so please fix the naming accordingly.

File: recipes/alphashape/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    - alphashape=alphashape.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - click >=6.0
+    - click_log >=0.3.2
+    - pip
+    - python
+    - scipy >=1.0.0
```
Comment (npavlovikj) [inline recipes/alphashape/meta.yaml]: Beside `pip` and `python`, are you sure you need the other dependencies for installing the package, and not only for running it? I would assume that you only need `pip` and `python` here...

File: recipes/alphashape/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - click_log >=0.3.2
+    - python
+    - scipy >=1.0.0
+    - shapely >=1.4.0
+
+test:
+  imports:
+    - alphashape
+  commands:
+    - python setup.py test
```
Comment (npavlovikj) [inline recipes/alphashape/meta.yaml]: The command you should test is the entrypoint you have created, `alphashape`.

File: recipes/alphashape/meta.yaml (hunk 2 above)
Comment (ngodber) [reply recipes/alphashape/meta.yaml]: fixed.

File: recipes/alphashape/meta.yaml (hunk 3 above)
Comment (ngodber) [reply recipes/alphashape/meta.yaml]: I agree, I misunderstood the difference between the two.

File: recipes/alphashape/LICENSE (hunk 1 above)
Comment (ngodber) [reply recipes/alphashape/LICENSE]: removed.

File: recipes/alphashape/meta.yaml (hunk 4 above)
Comment (ngodber) [reply recipes/alphashape/meta.yaml]: I've updated the cli test to alphashape --help. This returns the help text. Not sure if this is acceptable?

**Comment by npavlovikj** [comment]
The recipe LGTM, thank you for contributing @ngodber !

File: recipes/alphashape/meta.yaml (hunk 4 above)
Comment (npavlovikj) [reply recipes/alphashape/meta.yaml]: Yep, printing help text is fine.

**Pull request merged**